baselines/hfedxgboost/hfedxgboost/conf/clients/space_ga_2_clients.yaml

```diff
@@ -0,0 +1,9 @@
+n_estimators_client: 100
+num_rounds: 50
+client_num: 2
+num_iterations: 500
+
+xgb:
+  max_depth: 4
```

Comment:
Shouldn't be left as `8` (as indicated in the paper)?

Reply:
gone through that during our last meeting